packages/@o3r/configuration/schemas/configuration.metadata.schema.json

```
@@ -186,6 +186,13 @@
               "items": {
                 "type": "string"
               }
+            },
+            "restrictionKeys": {
+              "type": "array",
+              "description": "Restriction keys",
```

```suggestion
              "description": "Restriction keys to customize the user permissions",
```
just an idea, open to other suggestions